### PR TITLE
[RateLimiter] Return `getRetryAfter()` method usage

### PR DESCRIPTION
**Pull request by ERuban**
We cannot use `RateLimit->calculateTimeForTokens(1, 1)` there, bcs it is from `SlidingWindow` class

### CODE REVIEW

File: rate_limiter.rst [hunk 1]
```diff
@@ -353,7 +353,7 @@ the :class:`Symfony\\Component\\RateLimiter\\Reservation` object returned by the
             $limit = $limiter->consume();
             $headers = [
                 'X-RateLimit-Remaining' => $limit->getRemainingTokens(),
-                'X-RateLimit-Retry-After' => $limit->calculateTimeForTokens(1, 1),
+                'X-RateLimit-Retry-After' => $limit->getRetryAfter()->getTimestamp(),
```
Comment (wouterj) [inline rate_limiter.rst]: The header must contain the number of seconds from now to retry I believe. So we can do something like:

```suggestion
                'X-RateLimit-Retry-After' => now() - $limit->getRetryAfter()->getTimestamp(),
```

And then add `use function Symfony\Component\Clock\now;` at the top of the example.

**Review by wouterj — COMMENTED** [review]
You're 100% correct that we shouldn't use this (internal) method as public API.

**Comment by OskarStark** [comment]
I will take of the comments while merging, using `time()` over `now()` for now and switch 

```diff
- 'X-RateLimit-Retry-After' => now() - $limit->getRetryAfter()->getTimestamp(),
+ 'X-RateLimit-Retry-After' => $limit->getRetryAfter()->getTimestamp() - time(),
```

**Comment by OskarStark** [comment]
I am very sorry @ERuban I am not sure what happened, but my an error happened while merging your PR, I applied the change myself in e1e9205375955b6da8feda826ef6910bc0c44274

**Pull request closed without merging**